src/app/pages/signing/signing.component.html

```diff
@@ -67,17 +67,19 @@
         </button>
 
         <!-- Color pickers for text and background -->
-        <button type="button" class="btn btn-secondary btn-format mr-2" id="icon-wrapper">
-          <div id="font-color-text">A</div>
+        <button type="button" class="btn btn-secondary btn-format mr-2 icon-wrapper">
+          <span id="font-color-text">A</span>
+          <label for="colorpicker-font" hidden>Colour picker for font</label>
           <input type="color" class="mr-2" id="colorpicker-font" name="fontColorPicker" value={{textColor}}
             [(ngModel)]='textColor'>
-          <div class="dynamic-color-bar" [ngStyle]="{'background-color': textColor}">&nbsp;</div>
+          <span class="dynamic-color-bar" [ngStyle]="{'background-color': textColor}">&nbsp;</span>
         </button>
-        <button type="button" class="btn btn-secondary btn-format mr-2" id="icon-wrapper">
+        <button type="button" class="btn btn-secondary btn-format mr-2 icon-wrapper">
           <img id="icon" src="../../../assets/images/icons/SigningIcon_fill-color.svg">
```

Comment:
Still missing the alt attribute
```suggestion
          <img id="icon" src="../../../assets/images/icons/SigningIcon_fill-color.svg" alt="fill color selector">
```